vocab/security/vocabulary.drawio

```diff
@@ -237,42 +237,42 @@
+        <UserObject label="&lt;i&gt;domain&lt;/i&gt;" link="https://w3id.org/security#domain" id="Uf8WLKuzS3drS_BCJ-BJ-7">
           <mxCell style="rounded=1;whiteSpace=wrap;html=1;fontSize=16;fillColor=none;strokeWidth=2;" parent="1" vertex="1">
             <mxGeometry x="555.3489294710328" y="143" width="163.7314231738035" height="32.536500000000004" as="geometry" />
           </mxCell>
         </UserObject>
-        <UserObject label="&lt;i&gt;challenge&lt;/i&gt;" link="https://w3id.org/security/#challenge" id="Uf8WLKuzS3drS_BCJ-BJ-8">
+        <UserObject label="&lt;i&gt;challenge&lt;/i&gt;" link="https://w3id.org/security#challenge" id="Uf8WLKuzS3drS_BCJ-BJ-8">
           <mxCell style="rounded=1;whiteSpace=wrap;html=1;fontSize=16;fillColor=none;strokeWidth=2;" parent="1" vertex="1">
             <mxGeometry x="555.3489294710328" y="248" width="163.7314231738035" height="32.536500000000004" as="geometry" />
           </mxCell>
         </UserObject>
-        <UserObject label="&lt;i&gt;previousProof&lt;/i&gt;" link="https://w3id.org/security/#previousProof" id="Uf8WLKuzS3drS_BCJ-BJ-9">
+        <UserObject label="&lt;i&gt;previousProof&lt;/i&gt;" link="https://w3id.org/security#previousProof" id="Uf8WLKuzS3drS_BCJ-BJ-9">
           <mxCell style="rounded=1;whiteSpace=wrap;html=1;fontSize=16;fillColor=none;strokeWidth=2;" parent="1" vertex="1">
             <mxGeometry x="555.3489294710328" y="96.8925" width="163.7314231738035" height="32.536500000000004" as="geometry" />
           </mxCell>
         </UserObject>
-        <UserObject label="&lt;i&gt;proofPurpose&lt;br&gt;&lt;/i&gt;" link="https://w3.org/2018/credentials/#proofPurpose" id="Uf8WLKuzS3drS_BCJ-BJ-10">
+        <UserObject label="&lt;i&gt;proofPurpose&lt;br&gt;&lt;/i&gt;" link="https://w3id.org/security#proofPurpose" id="Uf8WLKuzS3drS_BCJ-BJ-10">
           <mxCell style="rounded=1;whiteSpace=wrap;html=1;fontSize=16;fillColor=none;strokeWidth=2;" parent="1" vertex="1">
             <mxGeometry x="555.3489294710328" y="299.15" width="163.7314231738035" height="32.536500000000004" as="geometry" />
           </mxCell>
         </UserObject>
-        <UserObject label="&lt;i&gt;proofValue&lt;/i&gt;" link="https://w3.org/2018/credentials/#proofValue" id="Uf8WLKuzS3drS_BCJ-BJ-11">
+        <UserObject label="&lt;i&gt;proofValue&lt;/i&gt;" link="https://w3id.org/security#proofValue" id="Uf8WLKuzS3drS_BCJ-BJ-11">
           <mxCell style="rounded=1;whiteSpace=wrap;html=1;fontSize=16;fillColor=none;strokeWidth=2;" parent="1" vertex="1">
             <mxGeometry x="555.3489294710328" y="440.29808823529413" width="163.7314231738035" height="32.536500000000004" as="geometry" />
           </mxCell>
         </UserObject>
-        <UserObject label="&lt;i&gt;expires&lt;/i&gt;" link="https://w3.org/2018/credentials/#expires" id="Uf8WLKuzS3drS_BCJ-BJ-12">
+        <UserObject label="&lt;i&gt;expires&lt;/i&gt;" link="https://w3id.org/security#expires" id="Uf8WLKuzS3drS_BCJ-BJ-12">
```

Comment:
This should be "expiration" ... but perhaps not for this PR. The context term is "expires" but the URL uses "#expiration". It's a unicorn, but has been in use for years that way.

```suggestion
        <UserObject label="&lt;i&gt;expires&lt;/i&gt;" link="https://w3id.org/security#expiration" id="Uf8WLKuzS3drS_BCJ-BJ-12">
```

Reply:
@dlongley Ah. So the term is `expires` in the spec, and, indeed, the `@context` uses `expiration` in the URL. Ouch.

There seems to be two ways to change this:

- Set the URL in the vocabulary to `expiration` rather than `expires` (the fact that the diagram uses `expires` is only an artifact). That means the vocabulary will be at odds with what the spec says, which is ugly.
- Update the `@context` file to the right ID

The second option seems to be the correct way of doing this. Any reason why we should not?

(But yes, I believe we should leave this for a separate PR.)

Reply:
Yes, the reason not to do it is that many things have shipped / escaped into the wild using `#expiration` in the URL. The contexts have been that way for a long while and we should probably just embrace it. The vocab was erroneously changed in some recent update that no one picked up on because it is a special unicorn -- and it should be changed back to `#expiration`, IMO. It's not pretty, but it is the way it's been done for a long time now and I don't think it buys us much to try and change to `#expires` -- with any number of possible issues coming up in the ecosystem if we do.

Reply:
I am holding my nose but... I will do it tomorrow. It is more than what you put there as a change request: the change must also be made in the svg file and, more importantly, in the vocabulary.yml file as well.

Reply:
@dlongley @msporny I have made the `expires -> expiration` change in https://github.com/w3c/vc-data-integrity/pull/212/commits/8afd6b9ff3a2cdfb7141843ad27511d8d5f1675d.

I was wondering whether we should put a small text as a comment into the vocabulary file explaining why, if one dereferences the definition link, one gets to the term `expires` in the VCDM spec. Because it seems to have a historical background, I would prefer one of you two to propose a text. Alternatively, we may remain silent...

(If you decide that a text is better, you can directly edit the yml file by adding a `comment:` field to the definition of `expiration`)